index.html

```diff
@@ -204,5 +204,19 @@ <h2 class="title">About Myself</h2>
             </ul>
             </div>
         </section>
+
+        <section class="contact">
+          <header>
+            <h2> Contact me </h2>
+            <p> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it  </p>
+          </header>
+      
+          <form action="https://formspree.io/f/mbjbnyye" method="post" class="contact-form">
+             <input type="text" name="username" placeholder="Enter Full Name" id="username" required>
+             <input type="email" name="user-email" placeholder="Enter e-mail" id="user-email" required>
```

Comment:
- [ ] Kindly make sure you add the validation regarding the length of the name, like recommended by the project requirements [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=Name%20field%20accepts%20up%20to%2030%20chars.)